README.md

```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow though steps in Contributing session.
```

Comment:
`Follow through the steps in [Contributing](#Contributing) section`

Reply:
Done, thanks Max!